return {
	["Play for 5 Minutes"] = {
		Type = "Typing",
		Description = "Play for 5 minutes",
		Goal = 300,
		Reward = 150,
		RewardType = "Coins",
	},
	["Win 5 Games"] = {
		Type = "Typing",
		Description = "Win 5 typing games",
		Goal = 5,
		Reward = 100,
		RewardType = "Coins",
	},
	["Type 500 Words"] = {
		Type = "Typing",
		Description = "Type 500 words correctly",
		Goal = 500,
		Reward = 199,
		RewardType = "Coins",
	},
	["Earn 3 Win Streak"] = {
		Type = "Typing",
		Description = "Get a win streak of 3",
		Goal = 3,
		Reward = 150,
		RewardType = "Coins",
	},
	["Complete Without Errors"] = {
		Type = "Typing",
		Description = "Complete a game with 0 errors",
		Goal = 1,
		Reward = 100,
		RewardType = "Coins",
	},
	["Purchase 3 Keyboards"] = {
		Type = "Shop",
		Description = "Buy 3 keyboards from the shop",
		Goal = 3,
		Reward = 150,
		RewardType = "Coins",
	},
}